src/extra/JsonSchemaRenderer/widgets/AvatarWidget.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { UiOption } from './ui-options';
+import { Widget, WidgetProps } from './widget-util';
+import { JsonTypes } from '../types';
+import { Avatar } from '../../../';
+
+const AvatarWidget: Widget = ({
+	value,
+	schema,
+	uiSchema,
+	...props
+}: WidgetProps) => {
+	console.log(value);
```

Comment:
console.log 🚓

Reply:
Feel free to self-certify once you fix this.

Reply:
removed ✂️